src/biz/template/ops.rs

```diff
@@ -245,7 +245,28 @@ pub async fn get_templates(
     None,
   )
   .await?;
-  Ok(templates)
+  let view_ids = templates.iter().map(|t| t.view_id).collect::<Vec<Uuid>>();
+  let publish_info_for_views =
+    select_published_collab_info_for_view_ids(pg_pool, &view_ids).await?;
+  let publish_info_map = publish_info_for_views
+    .into_iter()
+    .map(|info| (info.view_id, info))
+    .collect::<HashMap<Uuid, PublishInfo>>();
+  let templates_with_publish_info: Vec<TemplateMinimalWithPublishInfo> = templates
+    .into_iter()
+    .filter_map(|template| {
+      publish_info_map
+        .get(&template.view_id)
```

Comment:
Since `publish_info_map` is not being used afterwards, you could use 'remove' instead of `get` to own the `PublishInfo`, avoiding the need for 'clone()'